lib/spack/spack/cmd/config.py

```diff
@@ -117,7 +117,7 @@ def _get_scope_and_section(args):
     path = getattr(args, 'path', None)
 
     # w/no args and an active environment, point to env manifest
-    if not section:
+    if not section and not path:
```

Comment:
Shouldn't this be `and not scope`?

Reply:
Nope. I think this is followed in `spack config edit` s.t. it opens the spack.yaml file.

Reply:
changed that. not sure what `spack -e x config edit [path]` is supposed to do, but irrelevant for this pr.